pom.xml

```diff
@@ -301,10 +321,21 @@
         <jdbc-pool.version>7.0.81.wso2v2</jdbc-pool.version>
         <version.eclipse.ecj>4.4.2</version.eclipse.ecj>
         <version.com.google.code.gson>2.3.1</version.com.google.code.gson>
```

Comment:
Shall we update this to 2.7 please

Reply:
:+1: I will do it in a separate PR.

Reply:
Done in https://github.com/wso2/micro-integrator/pull/11